src/main/java/org/hyperledger/aries/api/credentials/CredentialAttributes.java

```diff
@@ -24,20 +24,14 @@
 import java.util.stream.Collectors;
 
 @Slf4j
-@Data @NoArgsConstructor @AllArgsConstructor @Builder
+@Data @Builder
```

Comment:
`@NoArgsConstructor` and `@AllArgsConstructor` are needed by Jackson, otherwise the class won't deserialise.